### DISCUSSION TITLE
Maizzle and Vite

### DISCUSSION THREAD

**Discussion by enguyen807** [maizzle/framework#1386] [Q&A] [answered]
I currently have an existing project that uses tailwindcss and Vite. Is there any support for Vite? It seems reductant for me to have both browsersync and vite in the same project.

**Comment by joshmanders** [comment, marked as answer]
Hey @enguyen807 I am working on a Vite plugin for Maizzle. @cossssmin and I are in contact about it and when it's ready we'll be moving it over into the official Maizzle org and maintenance.
